static/js/server_events_dispatch.js

```diff
@@ -752,6 +753,9 @@ export function dispatch_normal_event(event) {
             switch (event.op) {
                 case "add":
                     user_groups.add(event.group);
+                    if (overlays.groups_open()) {
+                        user_groups_settings_ui.add_group_to_table(event.group);
+                    }
```

Comment:
It might make sense as a follow-up to create a `user_group_events.js`, similar to `stream_events.js`, since over time this code path is going to get more complex.

Reply:
Yeah, I am thinking of adding `user_group_events.js` soon after adding commits for `add_members` and `remove_members`.

Reply:
Sounds good.